orchestrator/do-release/Jenkinsfile

```diff
@@ -62,20 +62,20 @@ def doRelease(exoUser, jenkinsAgentRootPath, jiraID, project, releaseCMD, isInPa
 }
 
 // Read Datas from JSON Catalog and executes releases
-def doReleases(exoUser, jenkinsAgentRootPath, jiraID, projectsToRelease, releaseCMD, isInParallel) {
+def doReleases(exoUser, jenkinsAgentRootPath, jiraID, projectsToRelease, releaseCMD, isInParallel, callback) {
```

Comment:
The logic looks good but I would rename some methods/variables to make the code more readable:
* doReleases -> applyOnReleasedProjects (since depending on the callback argument we do not necessarily do a release)
* callback -> processAction ? applyAction ? (because the word "callback" is banned when you did some javascript development ;) )

Reply:
The method name is changed to ``executeActionOnProjects`` and the needed parameters are now only the projects and the action